### PR TITLE
feat(orb-ui orb-docker): Dockerfile update npm to yarn

### PR DESCRIPTION
**Pull request by gpazuch**
Update [Dockerfile](./ui/docker/Dockerfile) to use yarn instead of npm.
Yarn resolves peer deps and package conflicts with more ease.
Also quicker than update the giant package.json dep list RN.

Update Orb UI dockerfile commands

### CODE REVIEW

**Review by daitan-rovaris — CHANGES_REQUESTED** [review]
Hey @gpazuch please add the RUN npm install -g yarn command, to be sure it will be there

File: ui/docker/Dockerfile [hunk 1]
```diff
@@ -2,9 +2,12 @@
 FROM node:14.15.4 as node
 WORKDIR /app
 COPY package.json /app/
-RUN npm install
+# I'm not sure if yarn is available from this node it imported
```
Comment (weyrick) [inline ui/docker/Dockerfile:5]: yes it's available, let's remove these comments and then we can land it

Comment (daitan-rovaris) [reply ui/docker/Dockerfile:5]: > yes it's available, let's remove these comments and then we can land it

:+1:

Comment (gpazuch) [reply ui/docker/Dockerfile]: You're right, it does indeed come pre-installed.
@daitan-rovaris I've just removed the --force;
After running both alternatives and checking yarn versions available, it seemed to be overkill.

**Review by daitan-rovaris — APPROVED** [review]
lgtm!

**Pull request merged**